configapp/static/js/utils.js

```diff
@@ -0,0 +1,41 @@
+function util_draw_polygons(canvas, ctx, image, polygons, occupies)
```

Comment:
@Adman I do not particularly like the naming here -- is the `unit_` part of the function's name necessary? Could a more descriptive name be used, perhaps?